tendrl/python_gdeploy/actions/gluster_package_installation.py

```diff
@@ -0,0 +1,38 @@
+from tendrl.python_gdeploy.wrapper import gdeploy_features as gf
+from tendrl.python_gdeploy.wrapper.gdeploy_wrapper import cook_gdeploy_config
+from tendrl.python_gdeploy.wrapper.gdeploy_wrapper import invoke_gdeploy
+
+GLUSTERFS_PACKAGES = [
+    "glusterfs",
+    "glusterfs-server",
+    "glusterfs-cli",
+    "glusterfs-libs",
+    "glusterfs-client-xlators",
+    "glusterfs-api",
+    "glusterfs-fuse"
+]
+
+GLUSTERFS_REPO = "https://buildlogs.centos.org/centos/7/"\
```

Comment:
Would it be same in case of downstream as well? dont think so. If yes it should be configurable ..